mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/SmartTestingMavenConfigurer.java

```diff
@@ -36,6 +40,13 @@ public void afterProjectsRead(MavenSession session) throws MavenExecutionExcepti
 
         configuration = Configuration.load();
 
+        if (session.getRequest().getLoggingLevel() == 0) {
+            enableMavenDebugLogLevel(true);
+            logger.debug("Maven build run in debug log level.");
```

Comment:
Do we really need this information?

Reply:
-1 from my side. I think we should remove it